src/frontend/shared-components/src/components/table/Table.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./Table.css";
+
+const TableElement = ({ element: { name, value } }) => {
+  return (
+    <tr>
+      <td>{name}</td>
+      <td>{value}</td>
+    </tr>
+  );
+};
+
+export default function Table({
+  table: { id, header, tableElements, tableStyle }
+}) {
+  const tableComponents = tableElements.map(element => {
+    return <TableElement key={element.key || element.name} element={element} />;
+  });
+
+  return (
+    <table id={id} className={tableStyle}>
+      <thead>
+        <tr>
+          <th colSpan="2">{header}</th>
+        </tr>
+      </thead>
+      <tbody>{tableComponents}</tbody>
+    </table>
+  );
+}
+
+TableElement.propTypes = {
+  element: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired
+  }).isRequired
+};
+
+Table.propTypes = {
+  table: PropTypes.shape({
+    id: PropTypes.string,
+    header: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
for `PropTypes.object` here do you mean jsx? If so, I think we should use `PropTypes.element`